spring-boot-starter-camunda-sdk/src/test/java/io/camunda/zeebe/spring/client/config/CredentialsProviderSelfManagedTest.java

```diff
@@ -53,13 +53,13 @@
       "camunda.client.mode=self-managed",
       "camunda.client.auth.client-id=my-client-id",
       "camunda.client.auth.client-secret=my-client-secret",
-      "camunda.client.auth.issuer=http://localhost:14682/auth-server"
+      "camunda.client.auth.issuer=http://localhost:14683/auth-server"
     })
 @EnableConfigurationProperties({
   ZeebeClientConfigurationProperties.class,
   CamundaClientProperties.class
 })
-@WireMockTest(httpPort = 14682)
+@WireMockTest(httpPort = 14683)
```

Comment:
🪄 ⚠️ I would prefer to avoid magic numbers here. Why?

Because, 

 1. We will likely run into the same issue again if someone else decides to use the same port for some test. We have for example already dynamic assignments for ports in other tests, what if they use this port and your test does the same? It is unclear and will fail.
 2. In general it is not clear why to use this port over others. It feels more like an bandaid.

In general, I would always advise to use a dynamic/random port that is free during the assignment. 

WireMock seems to also allow to use dynamic ports https://wiremock.org/docs/configuration/#network-ports-and-binding maybe something we want to look into.


Maybe some of these answers help as well https://stackoverflow.com/questions/48707625/set-property-with-wiremock-random-port-in-spring-boot-test

Reply:
Thanks for the advice, really nice one